core/trino-main/src/main/java/io/trino/sql/planner/SystemPartitioningHandle.java

```diff
@@ -261,7 +261,7 @@ public String toString()
             }
         }
 
-        private static class HashBucketFunction
+        public static class HashBucketFunction
```

Comment:
Also make it top level class so it's not hidden between private classes and other stuff